src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java

```diff
@@ -146,11 +147,17 @@ public void process(BlobClient blobClient) {
         }
     }
 
-    private byte[] tryToDownloadBlob(BlobClient blobClient) throws IOException {
+    private byte[] downloadBlob(BlobClient blobClient) throws IOException {
         try (var outputStream = new ByteArrayOutputStream()) {
             blobClient.download(outputStream);
 
             return outputStream.toByteArray();
+        } catch (Exception exc) {
+            String errorMessage = exc instanceof IOException
```

Comment:
can we not add a catch block for IOException ?

Reply:
If the condition is as simple as it currently is, sure. I initially made it more complex and it might end up like that.

Reply:
I think checking the contents of the exception message to see if it contains 502(or some string we which get back in the response when files are blocked by AV) that would be nicer